WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -1,52 +1,86 @@
 import UIKit
 
 protocol CompliancePopoverCoordinatorProtocol: AnyObject {
-    func presentIfNeeded(on viewController: UIViewController)
+    func presentIfNeeded()
     func navigateToSettings()
     func dismiss()
 }
 
 final class CompliancePopoverCoordinator: CompliancePopoverCoordinatorProtocol {
-    private weak var presentingViewController: UIViewController?
+
+    // MARK: - Dependencies
+
     private let complianceService = ComplianceLocationService()
     private let defaults: UserDefaults
 
+    // MARK: - Views
+
+    private static var window: UIWindow?
```

Comment:
Initially, this variable wasn't `static`, but I changed it to `static` for the following reason:

There is another bug where 2 `MySiteViewController` are created. You can verify this by following the **Test Instructions** steps and using the **Memory Graph** debugger.

So when 2 instances are created, 2 windows are also created and presented on screen. So to ensure this doesn't happened, I turned this variable `static`.

Reply:
I suppose it should be fine as we wouldn't show this multiple times anyhow. Although, would it cause a problem if we were to reuse the View for some other purpose?